plugins/GSdx/GSRenderer.cpp

```diff
@@ -496,24 +489,10 @@ void GSRenderer::VSync(int field)
 
 		m_snapshot.clear();
 	}
-	else
+	else if(m_dump)
 	{
-		if(m_dump)
-		{
-            bool control = false;
-
-            #ifdef _WIN32
-
-            control = !!(::GetAsyncKeyState(VK_CONTROL) & 0x8000);
-
-			#else
-
-			control = m_control_key;
-
-            #endif
-
-	    	m_dump.VSync(field, !control, m_regs);
-		}
+		if (m_dump->VSync(field, !m_control_key, m_regs))
```

Comment:
It is the first thing that I will do after the release. And I will likely do the core too.